texttospeech/ssml_addresses/tagged_resources/example.ssml

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Do the `tagged_resources` files need to be stored here, since they are basically duplicates that only hold the region tags?

Reply:
They're a little bit different. In the tutorial, I use short and long examples (see staged tutorial: https://cloud-dot-devsite.googleplex.com/text-to-speech/docs/ssml-tutorial). In resources, I just use the long example. Should I store them in resources but under a different name? I.e. 'tagged_example.txt'?

Reply:
I'd only keep what you use in the staged tutorial and everything else can be deleted.